readthedocs/doc_builder/python_environments.py

```diff
@@ -169,13 +169,23 @@ def install_core_requirements(self):
             *cmd, bin_path=self.venv_bin(), cwd=self.checkout_path
         )
 
-        requirements = [
-            'mock==1.0.1',
-            'pillow==5.4.1',
-            'alabaster>=0.7,<0.8,!=0.7.5',
-            'commonmark==0.8.1',
-            'recommonmark==0.5.0',
-        ]
+        requirements = []
```

Comment:
Looks like we can initialize this with the contents from https://github.com/readthedocs/readthedocs.org/pull/9473/files#diff-ae3dd3afeb422a62b42a209aa813ca5a60b9a22cf6c20f7bc3a1b1abf63a5148R181-R187

Reply:
I'm not sure where that link points exactly because it doesn't show me a specific line or similar.

However, I've done it in this way to avoid changing _all the tests_ that require a specific order of requirements to check them.

I want to merge and deploy this today, but if you feel strongly about this, we can continue talking and change it later.